test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -3624,4 +3770,75 @@ public int hashCode()
             return java.util.Objects.hash(user, protocolVersion, shouldUseEncryption, shouldUseCertificate);
         }
     }
+
+    protected PlanSelectionAssertion assertThatIndexQueryPlanFor(String query, Object[]... expectedRows)
+    {
+        // First execute the query capturing warnings and check the query results
+        disablePreparedReuseForTest();
+        ClientWarn.instance.captureWarnings();
+        assertRowsIgnoringOrder(execute(query), expectedRows);
+        List<String> warnings = ClientWarn.instance.getWarnings();
```

Comment:
Probably this and the two related methods (`warns` and `doesntWarn`) can be removed.